### ISSUE TITLE
Set up routing

### ISSUE THREAD

**Issue by jpham197** [jpham197/food-tracker#7]

**Comment by Khoi8** [comment]
What is the AC for this ticket?

**Comment by jpham197** [comment]
I don't think we need AC while we are still a fast moving startup working to complete our MVP. We should not be bogged down by these details.
